au.edu.uq.esys.escript.appdata.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Adam Ellery <a.ellery@uq.edu.au> -->
+<application>
+  <id type="desktop">au.edu.uq.esys.escript</id>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <name>esys-escript</name>
+  <summary> 
+    esys-escript is a python module for implementing mathematical models using the finite element method
+  </summary>
+  <description>
+    <p>
+      esys-escript is a programming tool for implementing mathematical models in python using the finite element method (FEM). As users do not access the data structures it is very easy to use and scripts can run on desktop computers as well as highly parallel supercomputer without changes. Scripts are executed in parallel using MPI, OpenMP and hybrid mode processing over 50 million unknowns on several thousand cores on a parallel computer.
+    </p>
+  </description>
+  <launchable type="desktop-id">au.edu.uq.esys.escript.desktop</launchable>
+  <url type="homepage">https://launchpad.net/escript-finley/</url>
+  <releases>
+    <release date="2019-01-01" version="5.3">
+      <description>
+        <p>
+          This release contains eScript version 5.3.1, which is based on svn revision 6823.
+        </p>
+      </description>
+    </release>
+  </releases>
+  <developer_name>University of Queensland</developer_name>
+  <screenshots>
+    <screenshot>
+      <image type="source">https://i.imgur.com/HwbUkV9.png</image>
```

Comment:
"The <screenshots/> tag contains multiple <screenshot/> children, where at least one of them must have the property type="default" to indicate the application's primary screenshot. Every <screenshot/> tag must have at least one <image/> child, which may define the width and height of the referenced image in it's properties. The value of the <image/> tag is a direct URL to a screenshot uploaded to a public location on the web. "
Per the specs here https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html
That's why appstream-util fails here I suppose.

Reply:
I've changed the appdata.xml file so that one of the screenshots has type="default".